ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,37 @@
+ages = [29, 19, 17, 26, 32, 16, 21]
+print(ages)
+
+print(ages[4])
+
+print(ages[-2])
+
+print(ages[2:6])
+
+print(ages[:3])
+
+if 17 in ages:
+  print(True)
+else:
+  print(False)
+
+if 42 in ages:
+  print(True)
+else:
+  print(False)
```

Comment:
We haven't really talked about conditionals yes, and you should be able to accomplish each of these in a single statement:

---

![image](https://user-images.githubusercontent.com/16143771/146114729-318914c4-3448-4aa5-8f5f-3007ddecd558.png)